placenames.mss

```diff
@@ -369,6 +369,38 @@
   }
 }
 
+#placenames-small::quarter {
+  [place = 'quarter'] {
+    [zoom >= 14][zoom < 17] {
+      text-name: "[name]";
+      text-fill: @placenames;
+      text-face-name: @book-fonts;
+      text-halo-fill: @standard-halo-fill;
+      text-halo-radius: @standard-halo-radius * 1.5;
+      [zoom >= 14] {
+        text-halo-fill: white;
+        text-size: 11;
+        text-wrap-width: 65; // 5.0 em
+        text-line-spacing: -0.65; // -0.05 em
```

Comment:
Indeed, `// -0.05 em` is reasonable for a text‑wrap‑width of _5.0 em_. The absolute value is than  _−0.05 em × 11 em⁻¹ = −0.55_